supportedBackends/tes/src/main/scala/cromwell/backend/impl/tes/TesJobPaths.scala

```diff
@@ -28,6 +28,15 @@ case class TesJobPaths private[tes] (override val workflowPaths: TesWorkflowPath
   val callInputsDockerRoot = callDockerRoot.resolve("inputs")
   val callInputsRoot = callRoot.resolve("inputs")
 
+  /*
+   * tesTaskRoot: This is the root directory that TES will use for files related to this task.
+   * We provide it to TES as a k/v pair where the key is "internal_path_prefix" (specified in TesWorkflowOptionKeys.scala)
+   * and the value is a blob path.
+   * This is not a standard TES feature, but rather related to the Azure TES implementation that Terra uses.
+   * While passing it outside of terra won't do any harm, we could consider making this optional and/or configurable.
+   */
+  val tesTaskRoot : Path = callExecutionRoot.resolve("tes_task")
```

Comment:
Does `resolve` throw an exception if it's not running on Azure TES? Or is it creating-if-not-exists?

Reply:
Good question, the latter.

Reply:
Good question. It requires `callExecutionRoot` to at least exist, but it will create `.../tes_task` if that doesn't yet exist. 

`callExecutionRoot` needs to exist in order to not throw, but that should, at the minimum, be an empty path at this point.